test/TMN.jl

```diff
@@ -233,4 +233,17 @@ set_variables(Interval{Float64}, [:x, :y], order=_order_max)
             "Interval(1.0, 1.0) y² + Interval(0.0, 0.0)"
     end
 
+    @testset "Integration" begin
```

Comment:
I think you are not testing here the integration of `TaylorModelN`, but the integration of `TaylorN` polynomials...